pkg/sensors/manager.go

```diff
@@ -11,7 +11,6 @@ import (
 	"github.com/cilium/tetragon/api/v1/tetragon"
 	"github.com/cilium/tetragon/pkg/k8s/apis/cilium.io/v1alpha1"
 	"github.com/cilium/tetragon/pkg/logger"
-	sttManager "github.com/cilium/tetragon/pkg/stt"
```

Comment:
I don't have the full context about what is `stt` but it seems that `pkg/stt/sttmanager.go` is not in use by any other code. Maybe we can remove `pkg/stt/sttmanager.go` as well in this commit?

Reply:
It relates to stack traces. Idea is to add them back at some point, so I thought maybe keep the code in the tree for now. We can always revisit this.